### DISCUSSION TITLE
How to enable red coloring of HTML tags in editor like the screenshot on the GitHub DAM main page?

### DISCUSSION THREAD

**Discussion by fabvit** [digao-dalpiaz/Dam#42] [Q&A] [answered]
How to enable red coloring of HTML tags in editor, like the screenshot on the GitHub DAM main page?
I am on Delphi Alexandria  and the HTML tags in the editor are in black like ordinary text in the editor

**Comment by digao-dalpiaz** [comment, marked as answer]
Hello,

The red tags in Editor form is from an old version, when the component only supported Delphi environment (using RichText component).
Now, as the component supports Lazarus and Delphi, it is not possible to use RichEdit in the editor, so the sintax highlight is not available anyomore.

**Comment by fabvit** [comment]
Hi Digao. 
Anyway thank you so much for such a useful component!


Il 18 maggio 2024 23:19:17 CEST, "Rodrigo Depiné Dalpiaz ***@***.***)" ***@***.***> ha scritto:
>
>ATTENZIONE: Questa email 
>è originata all’esterno di Envisioning. Non cliccare sui collegamenti né aprire gli allegati se non riconosci il mittente e sai che il contenuto è sicuro.
>
>
>Hello,
>
>The red tags in Editor form is from an old version, when the component only supported Delphi environment (using RichText component).
>Now, as the component supports Lazarus and Delphi, it is not possible to use RichEdit in the editor, so the sintax highlight is not available anyomore.
>
>
>-- 
>Reply to this email directly or view it on GitHub:
>https://github.com/digao-dalpiaz/Dam/discussions/42#discussioncomment-9482821
>You are receiving this because you authored the thread.
>
>Message ID: ***@***.***>

Le informazioni contenute nel presente documento sono riservate all'uso esclusivo del destinatario e possono essere soggette alla normativa sulla  privacy ed al segreto professionale. Se il lettore non e' l'inteso destinatario, si notifica che non deve utilizzarlo, riprodurlo,  diffonderlo, copiarlo in qualsiasi forma o agire facendovi affidamento. Se  Lei ha ricevuto questo messaggio per errore, e' pregato di distruggerlo, anche in ogni sua copia, ed avvisare immediatamente ENVISIONING al seguente numero +39 06-8632.5304 o all'indirizzo ***@***.*** La riservatezza delle informazioni e' tutelata indipendentemente da errori  nella trasmissione di questo messaggio.  L'integrita' e la sicurezza di questo messaggio non possono essere  garantite su Internet. Qualsiasi parere espresso in questo messaggio e' opera esclusiva del mittente e non rappresenta quello di ENVISIONING,  salvo quando il messaggio in modo specifico statuisce diversamente ed il  mittente e' autorizzato ad affermare che i pareri espressi sono di ENVISIONING. ENVISIONING S.R.L. si riserva il diritto di monitorare tutte le comunicazioni via e-mail effettuate tramite la propria rete. 
---
The information in this message is sent by a professional or His/Her agent, is intended to be confidential and for the use of only the individual or entity named above. If the reader of this message is not the intended recipient, You are notified that retention, dissemination, distribution or copying of this document is strictly prohibited. If you have received this message in error please delete it and any copies of it and notify ENVISIONING immediately at this number +39 06-8632.5304 or address ***@***.*** No confidentiality or privilege is waived or lost by any mistransmission. The integrity and safety of this message cannot be guaranteed on Internet. Any views expressed in this message are solely those of the individual sender and do not represent those of ENVISIONING, except where the message specifically states otherwise and the sender is authorized to  state them to be the views of ENVISIONING. ENVISIONING S.R.L. reserve the right to monitor all e-mail communications through its networks.
